config_ipa_test.go

```diff
@@ -0,0 +1,31 @@
+package verkle
+
+import (
+	"os"
+	"testing"
+)
+
+func TestGeneratePrecompFile(t *testing.T) {
```

Comment:
This test is double-checking that if the `precomp` file doesn't exist, it's generated with non-zero size.

The need for switching from `const` to `var` in the above file was necessary to be able to test the file generation case.
But things don't stop there, since `cfg` is also global; so there's some extra maneuvering to be sure other tests didn't already populated that variable.

Despite I agree `cfg` being global is somewhat pragmatic since clients don't need to hold `cfg` themselves, it makes some things a bit uglier on this side. From a pure personal taste perspective, I don't like globals and I'm usually fine asking clients to hold their own `cfg`.

In any case, there's no strict need to change this now. Maybe something to make all this safer is to have `GetConfg()` use a `sync.Once` since `GetConfig()` isn't thread-safe. While it's true that might be "part of the contract" that clients should call `GetConfig()` in `main` or something, you never know if some other dependency exists that also uses this package.

This is a bit of an offtopic PR discussion. Not saying we should do something now (or even later), it's not a big deal. Just sharing thoughts.

Reply:
thanks for sharing, yeah if other packages use this one, we might do that, but as long as no one requests this, there's no point in complicating the code for now.